### ISSUE TITLE
psc problem matcher

### ISSUE THREAD

**Issue by nwolverson** [nwolverson/vscode-ide-purescript#17]
Discussed #13; not clear how well this can match psc output format but certainly could be worth a try.

Downside of this is JSON compiler output as processed by PureScript specific build command can power quick-fix functionality, give metadata

**Comment by nwolverson** [comment]
Hmm didn't mean to close this

**Comment by minkir014** [comment]
Isn't is fixed by that commit. why don't you close it?

**Comment by henryxparker** [comment]
Is this issue referring to the output format of the json errors? If so, I would very much like to see it improve. Currently I cant use purescript IDE for errors with records because it is impossible to see subrecords clearly.

**Comment by nwolverson** [comment]
No, this referred to a problem matcher (vscode concept) to extract purs errors from the text-format output, rather than using the ide-purescript build command. 

JSON errors are formatted just as the regular compiler errors, as far as I know, and any issue with their formatting is an issue for the compiler repo unless it's specifially vcode related
